azavea/echo-locator
# Load dotenv file in CI/CD

## Overview

This PR:
- updates some frontend app code based on linter/formatter results
- updates the format script to also format frontend app code
- updates the bootstrap script to pull down `.env` file
- updates the `ci.yml` recipe to conditionally run `bootstrap` script


### Notes
1. I created a dev config bucket `echo-locator-devdjango-config-us-east-1 ` and uploaded the dotenv file for dev there.

## Testing Instructions

 * `AWS_PROFILE=<echo profile> ECHOLOCATOR_SETTINGS_BUCKET=echo-locator-devdjango-config-us-east-1 ./scripts/setup`
 * `./scripts/server`
 * The app frontend and backend should be built and run successfully
 * CI should also be happy


Closes https://github.com/azavea/echo-locator/issues/487
